tests/test_gcounter.py

```diff
@@ -0,0 +1,79 @@
+import uuid
+import random
+
+from crdt.gcounter import GCounter
+
+SEED = 42
+
+class TestGCounter():
+    """
+    Tests for the GCounter CRDT.
+    """
+
+    def randomGCounter(self):
+        counter = GCounter(id=uuid.uuid4())
+        for i in range(random.randint(0, 100)):
+            counter.add(random.randint(0, 100))
+        return counter
+
+    def test_counter(self):
+        """
+        Test that the basic operations work. This uses different IDs for the counters
+        to represent different nodes.
+        """
+        a = GCounter(id=uuid.uuid4())
+        a.add(1)
+        a.add(2)
+        a.add(3)
+        assert a.get() == 6
+
+        b = GCounter(id=uuid.uuid4())
+        b.add(4)
+        assert b.get() == 4
+
+        c = GCounter(id=uuid.uuid4())
+        c.add(10)
+        assert c.get() == 10
+
+        assert a.merge(b).get() == 10
```

Comment:
Maybe we could add a test after this one to show that `a` is also aware of `b` values after the merge (and potentially a test before to show that `a` is not aware of `b` values before the merge)